modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,602 @@
+        found = nullptr;
+
+        module->removeTargetBrowserPair(id);
+
+        // Remove from engine
+        openspace::global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + browser + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+
+        openspace::global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + target + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+    }
+    
+
+    return 0;
+}
+
+int translateScreenSpaceRenderable(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 5, "lua::translateScreenSpaceRenderable");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    float startX = ghoul::lua::value<float>(L, 1);
+    float startY = ghoul::lua::value<float>(L, 1);
+    float transX = ghoul::lua::value<float>(L, 1);
+    float transY = ghoul::lua::value<float>(L, 1);
+    
+    ScreenSpaceRenderable* renderable = global::renderEngine->screenSpaceRenderable(id);
+    renderable->translate(glm::vec2(transX, transY), glm::vec2(startX, startY));
+
+    return 0;
+}
+
+int removeSelectedImageInBrowser(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::removeSelectedImageInBrowser");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    // Image index
+    const int i = ghoul::lua::value<int>(L, 1);
+    // Get browser
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    const ImageData& image = module->getWwtDataHandler()->getImage(i);
+        
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->removeSelectedImage(i);
+    }
+
+    return 0;
+}
+
+int setEquatorialAim(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 3, "lua::setEquatorialAim");
+    // Browser id
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    double ra = ghoul::lua::value<double>(L, 1);
+    double dec = ghoul::lua::value<double>(L, 1);
+
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setEquatorialAim(glm::dvec2(ra, dec));
+    }
+
+    return 0;
+}
+
+int setVerticalFov(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 2, "lua::setVerticalFov");
+    // Browser id
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    float vfov = ghoul::lua::value<float>(L, 1);
+
+    // Get module
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    TargetBrowserPair* pair = module->getPair(id);
+    if (pair) {
+        pair->setVerticalFov(vfov);
+    }
+
+    return 0;
+}
+
+int setBorderColor(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 4, "lua::setBorderColor");
+    // Browser id
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
```

Comment:
and here;  both for:
`auto [id, r, g, b] = ghoul::lua::values<std::string, int, int, int>(L);`
or
`auto [id, color] = ghoul::lua::values<std::string, glm::ivec3>(L);`
and only taking two arguments